demisto_sdk/commands/content_graph/objects/base_content.py

```diff
@@ -202,7 +202,11 @@ def _save(
         """
         for key, val in self.field_mapping.items():
             attr = getattr(self, key)
-            if key == "marketplaces":
+            if key == "docker_image":
+                attr = str(attr)
+            elif key in ["params"]:
+                continue
```

Comment:
why?
we can write params as well

Reply:
It's being handled under integration, we can save params because it's an object we need to turn to dict and remove nulls.